### PR TITLE
Remove precision fns

### PR DESCRIPTION
**Pull request by ajsantander**

### CODE REVIEW

File: packages/synthetix-main/contracts/storage/Distribution.sol [hunk 1]
```diff
@@ -238,16 +232,16 @@ library Distribution {
      */
     function getActorValueChange(Data storage dist, bytes32 actorId) internal view returns (int valueChange) {
         DistributionActor.Data storage actor = dist.actorInfo[actorId];
-        int128 deltaValuePerShare = dist.valuePerShare - actor.lastValuePerShare;
+        int128 deltaValuePerShare = dist.valuePerShareD27 - actor.lastValuePerShareD27;
 
-        int changedValueHighPrecision = deltaValuePerShare * actor.shares.uint128toInt256();
-        valueChange = changedValueHighPrecision.fromHighPrecisionDecimalToInteger();
+        int changedValueHighPrecision = deltaValuePerShare * actor.sharesD18.uint128toInt256();
```
Comment (leomassazza) [inline packages/synthetix-main/contracts/storage/Distribution.sol:237]: mixing D27 and D18 here. Check if maths work

File: packages/synthetix-main/contracts/storage/DistributionEntry.sol [hunk 2]
```diff
@@ -83,14 +83,14 @@ library DistributionEntry {
 
         // determine whether this is an instant distribution or a delayed distribution
         if (entry.duration == 0 && entry.lastUpdate < entry.start) {
-            valuePerShareChange = (int(entry.scheduledValue) * 1e18) / int(totalSharesAmount);
+            valuePerShareChange = (int(entry.scheduledValueD18) * 1e18) / int(totalSharesAmount);
```
Comment (leomassazza) [inline packages/synthetix-main/contracts/storage/DistributionEntry.sol:86]: scheduledValueD18.divideDecimal(int(totalSharesAmount)) ?

File: packages/synthetix-main/contracts/storage/Pool.sol [hunk 3]
```diff
@@ -219,10 +219,10 @@ library Pool {
             thing = int(DecimalMath.UNIT); // If minLiquidityRatio is zero, then TODO
```
Comment (leomassazza) [inline packages/synthetix-main/contracts/storage/Pool.sol:219]: thing... really? LoL

File: packages/synthetix-main/contracts/storage/Distribution.sol (hunk 1 above)
Comment (ajsantander) [reply packages/synthetix-main/contracts/storage/Distribution.sol:237]: Yep, it goes to D45, then to D18 on the next line

File: packages/synthetix-main/contracts/storage/Pool.sol (hunk 3 above)
Comment (ajsantander) [reply packages/synthetix-main/contracts/storage/Pool.sol]: Yes, this has to do with the "passes" strategy. Seems like during the last pass I identified that to be a thing, but not what it is. We can probably rename it properly on the next pass.

**Pull request merged**